prereise/gather/griddata/hifld/const.py

```diff
@@ -385,3 +385,17 @@
     "Solar Thermal with Energy Storage": 0,
     "Solar Thermal without Energy Storage": 0,
 }
+
+dc_line_ratings = {  # MW
+    108354: 500,  # Square Butte
+    113313: 660,  # Neptune Cable
+    131914: 2000,  # Quebec – New England Transmission (Ayer to Monroe)
+    150123: 1000,  # CU
+    157627: 330,  # Cross-Sound Cable
+    157629: 660,  # Hudson Project
+    158515: 2000,  # Quebec – New England Transmission (Quebec to Monroe)
+    200823: 3100,  # Pacific DC Intertie (Path 65)
+    308464: 2400,  # Intermountain Power Project (Path 27)
+    310053: 400,  # Trans-Bay Cable
+    311958: 5,  # Alamogordo Solar Energy Center
+}
```

Comment:
Let's put down the data source of these references somewhere in the doc string.

Reply:
The lines themselves were identified based on personal judgement (substation names, geographical paths), and the capacities were compiled based on variety of sources. I'm not sure how to really cite that.

Reply:
Just wondering, since personal judgment is involved, should we create a DC line file with the needed information that would be loaded and remove all DC lines from the transmission file?

Reply:
I think this way seems more transparent: we intake raw un-processed external data, then apply the minimum amount of assumptions, made in the open.

Reply:
Maybe we can just put down this line for future references:
`These DC lines are identified based on substation names, geographical paths, etc. manually, the capacities of which are compiled based on public accessible sources.`?

Reply:
Good idea, I just pushed something to that effect.